File: Components/ASP.NET/VB/Documents/UserControls/SaveAsDocumentDialog.ascx.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports DevExpress.Web

Partial Public Class UserControls_SaveAsDocumentDialog
    Inherits System.Web.UI.UserControl

    Public Property DefaultDocumentName() As String

    Protected Sub Page_Load(ByVal sender As Object, ByVal e As EventArgs)
        DocumentNameTextBox.Text = DefaultDocumentName
    End Sub
End Class

File: Components/ASP.NET/CS/Documents/UserControls/SaveAsDocumentDialog.ascx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;
using DevExpress.Web;

public partial class UserControls_SaveAsDocumentDialog : System.Web.UI.UserControl {
    public string DefaultDocumentName { get; set; }

    protected void Page_Load(object sender, EventArgs e) {
        DocumentNameTextBox.Text = DefaultDocumentName;
    }
}
